requirements/base.txt

```diff
@@ -29,4 +29,4 @@ google-api-python-client==1.5.1
 -e git+https://github.com/openfun/ProctorU-xBlock.git@fun-1.1.2#egg=proctoru-xblock
 
 # Glowbl
--e git+https://github.com/openfun/xblock-Glowbl.git@743561fb77b1b3c0ba9aaae0fdca86c3204c1ffd#egg=fun-glowbl-xblock
+-e git+https://github.com/openfun/xblock-Glowbl.git@77c7c1d187d770bc800c217d97f290d9f8c3c8ca#egg=fun-glowbl-xblock
```

Comment:
Why not using the release (git tag instead)?

https://github.com/openfun/xblock-Glowbl/blob/master/setup.cfg#L6